charts/envoy/values.yaml

```diff
@@ -61,13 +61,13 @@ service:
       targetPort: 50051
       # service.ports.envoy.protocol -- envoy protocol
       protocol: TCP
-    envoy-priv:
-      # service.ports.envoy-priv.port -- nvoy public port
-      port: 50052
-      # service.ports.envoy-priv.targetPort -- envoy k8s internal name
-      targetPort: 50052
-      # service.ports.envoy-priv.protocol -- envoy protocol
-      protocol: TCP
+  # envoy-priv:
+  #   # service.ports.envoy-priv.port -- envoy public port
+  #   port: 50052
+  #   # service.ports.envoy-priv.targetPort -- envoy k8s internal name
+  #   targetPort: 50052
+  #   # service.ports.envoy-priv.protocol -- envoy protocol
+  #   protocol: TCP
```

Comment:
I commented out these inappropriate default values instead of removing them.
This is because we can show that the Envoy chart can accept several port settings as a sample for users.

Basically, we should set (override/add) several port in another chart (e.g. Scalar DL) instead of setting them as default values in Envoy chart.